packages/react-instantsearch-theme-algolia/styles/_Panel.scss

```diff
@@ -0,0 +1,10 @@
+.ais-Panel__empty {
+  display: none;
```

Comment:
For a first iteration I would prefer to provide a "disabled" styling to the content of the panel (to indicate it's empty) rather than hiding it completely. What do you think?

We could also think we designers to understand if there's a preferred current way to indicate a widget cannot be used anymore.

I am proposing this because on some widgets it may makes less sense to hide everything:
- slider
- range
- multi ranges